### PR TITLE
moved empty-entry.js from project root

### PR DESCRIPTION
**Pull request by zouhir**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const defaultOptions = {
 
   ServiceWorker: {
     output: 'sw.js',
-    entry: path.join(__dirname, '../empty-entry.js'),
+    entry: path.join(__dirname, '../src/empty-entry.js'),
```
Comment (NekR) [inline src/index.js]: `entry: path.join(__dirname, './empty-entry.js'),`

**Comment by NekR** [comment]
Build doesn't fail with your way because you have `src/` dir. When it's installed with `npm` `src/` isn't shipped, so there will an error.

**Comment by zouhir** [comment]
ah! my first attempt was to do it your way `path.join(__dirname, './empty-entry.js')` but tests failed at `Comparing: basic-wIth-sw-out` - I'll look again

**Comment by NekR** [comment]
hmm.. maybe babel doesn't generate empty files?

**Comment by NekR** [comment]
Maybe just put it into `tpls/`?

**Comment by zouhir** [comment]
sh!t!! it does but adds `"use strict";` to it! that's why test's are failing

**Comment by zouhir** [comment]
sorry just saw your comment - yep I can do in `tpls/`

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "lib/",
     "tpls/",
     "runtime.js",
-    "empty-entry.js",
+    "src/empty-entry.js",
```
Comment (NekR) [inline package.json]: Remove this and we are done :-)

**Comment by NekR** [comment]
Oh, also this
image

**Pull request merged**